batcher/aligned-sdk/src/utils.rs

```diff
@@ -0,0 +1,17 @@
+use crate::errors::SubmitError;
+use crate::models::ProvingSystemId;
+
+pub fn parse_proving_system(proving_system: &str) -> Result<Option<ProvingSystemId>, SubmitError> {
+    match proving_system {
+        "GnarkPlonkBls12_381" => Ok(Some(ProvingSystemId::GnarkPlonkBls12_381)),
+        "GnarkPlonkBn254" => Ok(Some(ProvingSystemId::GnarkPlonkBn254)),
+        "Groth16Bn254" => Ok(Some(ProvingSystemId::Groth16Bn254)),
+        "SP1" => Ok(Some(ProvingSystemId::SP1)),
+        "Halo2IPA" => Ok(Some(ProvingSystemId::Halo2IPA)),
+        "Halo2KZG" => Ok(Some(ProvingSystemId::Halo2KZG)),
+        _ => Err(SubmitError::InvalidProvingSystem(
+            proving_system.to_string(),
+        )),
+        // _ => Err(anyhow!("Invalid proving system: {}, Available proving systems are: [GnarkPlonkBls12_381, GnarkPlonkBn254, Groth16Bn254, SP1, Halo2KZG, Halo2IPA]", proving_system))
```

Comment:
Remove unused comment 
```suggestion
```

Reply:
Solved in [614dec3](https://github.com/yetanotherco/aligned_layer/pull/459/commits/614dec3df6073efc6bc14d543db050ffe8ea00c7)